target/linux/mvebu/patches-4.19/560-helios4-dts-status-led-alias.patch

```diff
@@ -0,0 +1,28 @@
+--- a/arch/arm/boot/dts/armada-388-helios4.dts
++++ b/arch/arm/boot/dts/armada-388-helios4.dts
+@@ -15,6 +15,13 @@
+ 	model = "Helios4";
+ 	compatible = "kobol,helios4", "marvell,armada388",
+ 		"marvell,armada385", "marvell,armada380";
++		
++	aliases {
++		led-boot = &led_power;
++		led-failsafe = &led_power;
++		led-running = &led_power;
++		led-upgrade = &led_power;
++	};
+ 
+ 	memory {
+ 		device_type = "memory";
+@@ -70,10 +77,9 @@
+ 
+ 	system-leds {
+ 		compatible = "gpio-leds";
+-		status-led {
```

Comment:
It's a nitpick, but I'd keep the designation "status" here:
`led_status: status-led {`

Reply:
I copied how it was done for other mvebu devices https://github.com/openwrt/openwrt/blob/master/target/linux/mvebu/patches-5.4/240-linksys-status-led.patch

but ok, can change that

Reply:
In the references, the LED has been named power beforehand, and that is kept. In your case, the LED is called "status" beforehand, and that should be kept.